NewSpring/corinth
# fix: Adds Pills Back to Universal Content Items

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR adds the pills back to `UniversalContentItem` items.

### How do I test this PR?
Start up the sim and make sure that items like news, articles, and stories have a pill on their card in the feed.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
